lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -325,16 +337,15 @@ def run
 
     return if get_config(:mark_as_template)
     if get_config(:power) || get_config(:bootstrap)
-      vm = find_in_folder(dest_folder, RbVmomi::VIM::VirtualMachine, vmname) ||
+      @vm ||= find_in_folder(dest_folder, RbVmomi::VIM::VirtualMachine, vmname) ||
            fatal_exit("VM #{vmname} not found")
-      vm.PowerOnVM_Task.wait_for_completion
+      @vm.PowerOnVM_Task.wait_for_completion
       puts "Powered on virtual machine #{vmname}"
     end
 
     return unless get_config(:bootstrap)
-    sleep 2 until vm.guest.ipAddress
+    sleep 2 until connect_host
```

Comment:
This sleep may be unnecessary as we sleep in the connect_host and ipv4_address methods.